proto/reaction.proto

```diff
@@ -304,11 +333,42 @@ message Vessel {
       CUSTOM = 1;
       NONE = 2;
       OVEN_DRIED = 3;
+      FLAME_DRIED = 4;
+      EVACUATED_BACKFILLED = 5;
+      PURGED = 6;
     }
   }
-  VesselPreparation.VesselPreparationType preparation = 5;
+  repeated VesselPreparation.VesselPreparationType preparation = 5;
```

Comment:
@skearnes what is the right norm for singular/plural with repeated fields? Should it be `preparation` or `preparations`? The [examples](https://developers.google.com/protocol-buffers/docs/proto3) all use plural names for a repeated field, so I'm leaning that way.

Reply:
Done.